File: au-association-go/model/system/Activity.go
package system

import "time"

type Activity struct {
	Id          string    `json:"id" gorm:"primary_key"`
	Name        string    `json:"name"`
	Assid       int       `json:"assid"`
	Date        time.Time `json:"date"`
	Step        int       `json:"step"`
	File1       string    `json:"file1"`
	File2       string    `json:"file2"`
	Backmsg     string    `json:"backmsg"`
	Description string    `json:"description"`
	CreatedAt   time.Time `gorm:"column:create_time" json:"createTime"`
	UpdatedAt   time.Time `gorm:"column:update_time" json:"updateTime"`
}

func (Activity) TableName() string {
	return "activity"
}

File: au-java/src/main/java/com/fran6k/study/springbootstudy/bean/InReviewBean/Activity.java
package com.fran6k.study.springbootstudy.bean.InReviewBean;

import lombok.Data;
import org.hibernate.annotations.GenericGenerator;
import org.springframework.data.annotation.CreatedDate;
import org.springframework.data.annotation.LastModifiedDate;
import org.springframework.data.jpa.domain.support.AuditingEntityListener;

import javax.persistence.*;
import java.util.Date;

@Entity
@Table(name = "activity")
@Data
@EntityListeners(AuditingEntityListener.class)

public class Activity {
    @Id
    @GeneratedValue(generator = "paymentableGenerator")
    @GenericGenerator(name = "paymentableGenerator", strategy = "uuid")
    private String id;

    @Column(name = "name")
    private String name;


    @Column(name = "assid")
    private int assid;

    @Column(name = "date")
    private Date date;

    @Column(name = "step")
    private int step;

    @Column(name = "file1")
    private String file1;

    @Column(name = "file2")
    private String file2;

    @Column(name = "backmsg")
    private String backmsg;

    @Column(name = "description")
    private String description;

    @CreatedDate//自动添加创建时间的注解
    private Date createTime;
    @LastModifiedDate//自动添加更新时间的注解
    private Date updateTime;


}
